Source/x11/XGServerWindow.m

```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
     case GSIBeamCursor:
       cursor = XCreateFontCursor(dpy, XC_xterm);
       break;
-    case GSCrosshairCursor:
-      cursor = XCreateFontCursor(dpy, XC_crosshair);
+    case GSClosedHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_fleur);
       break;
-    case GSDisappearingItemCursor:
-      cursor = XCreateFontCursor(dpy, XC_shuttle);
+    case GSOpenHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_hand1);
       break;
     case GSPointingHandCursor:
-      cursor = XCreateFontCursor(dpy, XC_hand1);
+      cursor = XCreateFontCursor(dpy, XC_hand2);
+      break;
+    case GSCrosshairCursor:
+      cursor = XCreateFontCursor(dpy, XC_crosshair);
       break;
     case GSResizeDownCursor:
-      cursor = XCreateFontCursor(dpy, XC_bottom_side);
+      cursor = XCreateFontCursor(dpy, XC_sb_down_arrow);
```

Comment:
Sorry, but for all this resize images I prefer the old definitions. I might be old fashioned but this is what I expect for resize.

Reply:
I've reverted these changes.